build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -13,14 +13,14 @@ namespace Microsoft.DotNet.Cli.Build
     public class DebTargets
     {
         [Target(nameof(GenerateSdkDeb))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
I chatted with @brthor already. There is work that we will need to do in the repo to get that working. I don't know 100% the nature of that work. But instead of blocking having at least a tarball for ubuntu 16.04 on having a deb package for it, I thought it was better to get the tarball .